web-api/src/mind.rs

```diff
@@ -300,8 +302,8 @@ fn ndcg(relevance: &[f32], k: &[usize]) -> Vec<f32> {
         .collect()
 }
 
-fn main() {
-    if let Err(e) = run_benchmark() {
+async fn main() {
+    if let Err(e) = run_benchmark().await {
```

Comment:
:wrench: the compilers is unhappy because it can't find the `run_bechmark()` anymore, which was renamed. you could also remove the `main()` function here, as this is going to be resolved with #737 anyways.

Reply:
removed the `main`